Modules/IO/NIFTI/src/itkNiftiImageIO.cxx

```diff
@@ -1886,32 +1885,46 @@ NiftiImageIO::SetImageIOOrientationFromNIfTI(unsigned short dims, double spacing
         this->GetLegacyAnalyze75Mode() != NiftiImageIOEnums::Analyze75Flavor::AnalyzeITK4 &&
         this->GetLegacyAnalyze75Mode() != NiftiImageIOEnums::Analyze75Flavor::AnalyzeITK4Warning)
     { // only do this for Analyze file format
-      SpatialOrientationAdapter::OrientationType orient;
+      DICOMOrientation orient(DICOMOrientation::OrientationEnum::INVALID);
       switch (this->m_NiftiImage->analyze75_orient)
       {
         case a75_transverse_unflipped:
-          orient = SpatialOrientationEnums::ValidCoordinateOrientations::ITK_COORDINATE_ORIENTATION_RPI;
+          orient = DICOMOrientation(DICOMOrientation::CoordinateEnum::RightToLeft,
```

Comment:
In these long and and long-ish statements, it might make sense to `using CE = DICOMOrientation::CoordinateEnum`, then use `CE::RightToLeft, CE::PosteriorToAnterior, CE::InferiorToSuperior` on a single line. This applies to many places in the code, including previous commits.